### DISCUSSION TITLE
Terncy DC01 not pairing

### DISCUSSION THREAD

**Discussion by DubhAd** [Koenkk/zigbee2mqtt#7259] [Devices]
Quite possibly related to [this problem](https://github.com/Koenkk/zigbee2mqtt/discussions/5682#discussioncomment-389018) with the SD01, but I can't get my new DC01 paired to my ZZH, running the 2021 firmware (Z2M 1.18.1). I've tried where I want to install it (near a few router devices) and by the controller - no difference.

I'm running on channel 20 for compatibility with my Konke devices.

Is there anybody out there who's managed to get this paired to anything CC2652 based?

**Comment by DubhAd** [comment]
So, it _still_ won't pair to my ZZH. I've updated Z2M (currently 1.21.0-dev to deal with the fact that 1.21.0 won't let me pair anything) and the stick firmware (to 20210708). Nothing.

Spin up another container with a CC2531 and it (and any other Terncy device) pairs instantly...

So, three possibilities.

1. The Terncy devices don't like channel 20 (seems unlikely, but who knows)
2. The Terncy devices and CC2652 don't get on
3. There's something messed up with my mesh 😬

**Reply by Koenkk** [reply]
Does it pair when holding the device very close to the CC2652?

**Reply by DubhAd** [reply]
I've tried about 30cm away, yet with the CC2531 it pairs instantly about 3 meters away.

I'll try a couple of cm away from the antenna tomorrow.

**Comment by DubhAd** [comment]
Tried it, their SD-01 rotary controller, and their PP-01 "awareness switch". All pair instantly to the CC2531 anywhere in the room. None pair at all to the CC2652 at any distance.

I tried at all distances from touching the plastic case around the CC2652, practically touching the antenna, out to about 5 cm in 1 cm steps, and then again at about 15 cm and 30 cm, as well as on my desk (about 2 meters away).

**Comment by DubhAd** [comment]
Debug log shows ... _nothing about it_

Didn't show up at all. I don't know if that means it doesn't support channel 20, or if there's something else weird going on. I will see about getting a hold of a spare CC2652 based coordinator and using that in my test environment - but clearly that won't be quick 😁

**Comment by DubhAd** [comment]
Problem was solved... by changing the coordinator.

It paired immediately with my Sonoff CC2652 based coordinator 🤷 Same firmware version on the ZZH gets nothing.

Gremlins...

**Comment by Yuke-hd** [comment]
I'm using a sonoff dongle-E and my DC01 also won't pair, could it be a zigbee channel issue? I'm currently on channel 20, which channel worked for you? I know some zigbee devices are picky with the channels (Eg. aqara FP1)

**Reply by Yuke-hd** [reply]
Okay... here's my findings 
Tried a Conbee II stick with deconz, the default zigbee channel was 11 and it won't pair
Set the zigbee channel to 14 and Voilà it got paired. 

We now can confirm DC01 *at leaset* won't work with channel 11(deconz) and 20(EFR32MG21)
<img width="649" alt="image" src="https://github.com/Koenkk/zigbee2mqtt/assets/38873367/b0758cad-5b74-433f-b153-963512247dc1">

**Comment by Yuke-hd** [comment]
Ok I did more tests and can confirm this device won't work on channel >= 20 [similar to Aqara FP1](https://www.zigbee2mqtt.io/devices/RTCZCGQ11LM.html#channel)
